<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Tile</class>
 <widget class="QMainWindow" name="Pyatnashki">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>340</width>
    <height>280</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>340</width>
    <height>280</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>340</width>
    <height>280</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Tile</string>
  </property>
  <property name="toolTip">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>16777213</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>DejaVu Sans</family>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="text">
     <string>1</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>0</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DejaVu Sans</family>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>0</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DejaVu Sans</family>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>0</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DejaVu Sans</family>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DejaVu Sans</family>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>60</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DejaVu Sans</family>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>60</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DejaVu Sans</family>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_8">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>60</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DejaVu Sans</family>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_9">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>120</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DejaVu Sans</family>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_10">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>120</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DejaVu Sans</family>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="text">
     <string>10</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_11">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>120</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DejaVu Sans</family>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="text">
     <string>11</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_12">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>120</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DejaVu Sans</family>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="text">
     <string>12</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_13">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>180</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DejaVu Sans</family>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="text">
     <string>13</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_14">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>180</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DejaVu Sans</family>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="text">
     <string>14</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_15">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>180</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DejaVu Sans</family>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="text">
     <string>15</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_16">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>180</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DejaVu Sans</family>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="text">
     <string>16</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>0</y>
      <width>20</width>
      <height>240</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="moves">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>40</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>14</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
      <strikeout>false</strikeout>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="moves_title">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>10</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>14</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
      <strikeout>false</strikeout>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="text">
     <string>Moves:</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>340</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuGame">
    <property name="title">
     <string>Game</string>
    </property>
    <addaction name="actionNew_Game"/>
    <addaction name="actionReset"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionHelp"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuGame"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNew_Game">
   <property name="text">
    <string>New Game</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionReset">
   <property name="text">
    <string>Reset</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>Help</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>pushButton</tabstop>
  <tabstop>pushButton_2</tabstop>
  <tabstop>pushButton_3</tabstop>
  <tabstop>pushButton_4</tabstop>
  <tabstop>pushButton_5</tabstop>
  <tabstop>pushButton_6</tabstop>
  <tabstop>pushButton_7</tabstop>
  <tabstop>pushButton_8</tabstop>
  <tabstop>pushButton_9</tabstop>
  <tabstop>pushButton_10</tabstop>
  <tabstop>pushButton_11</tabstop>
  <tabstop>pushButton_12</tabstop>
  <tabstop>pushButton_13</tabstop>
  <tabstop>pushButton_14</tabstop>
  <tabstop>pushButton_15</tabstop>
  <tabstop>pushButton_16</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>Pyatnashki</receiver>
   <slot>clickPushButton()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>40</x>
     <y>23</y>
    </hint>
    <hint type="destinationlabel">
     <x>373</x>
     <y>34</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>Pyatnashki</receiver>
   <slot>clickPushButton()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>109</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>343</x>
     <y>47</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>Pyatnashki</receiver>
   <slot>clickPushButton()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>156</x>
     <y>15</y>
    </hint>
    <hint type="destinationlabel">
     <x>371</x>
     <y>19</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_4</sender>
   <signal>clicked()</signal>
   <receiver>Pyatnashki</receiver>
   <slot>clickPushButton()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>236</x>
     <y>21</y>
    </hint>
    <hint type="destinationlabel">
     <x>357</x>
     <y>29</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_5</sender>
   <signal>clicked()</signal>
   <receiver>Pyatnashki</receiver>
   <slot>clickPushButton()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>46</x>
     <y>100</y>
    </hint>
    <hint type="destinationlabel">
     <x>340</x>
     <y>91</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_6</sender>
   <signal>clicked()</signal>
   <receiver>Pyatnashki</receiver>
   <slot>clickPushButton()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>116</x>
     <y>75</y>
    </hint>
    <hint type="destinationlabel">
     <x>369</x>
     <y>136</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_7</sender>
   <signal>clicked()</signal>
   <receiver>Pyatnashki</receiver>
   <slot>clickPushButton()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>165</x>
     <y>71</y>
    </hint>
    <hint type="destinationlabel">
     <x>356</x>
     <y>81</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_8</sender>
   <signal>clicked()</signal>
   <receiver>Pyatnashki</receiver>
   <slot>clickPushButton()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>239</x>
     <y>110</y>
    </hint>
    <hint type="destinationlabel">
     <x>396</x>
     <y>107</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_9</sender>
   <signal>clicked()</signal>
   <receiver>Pyatnashki</receiver>
   <slot>clickPushButton()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>38</x>
     <y>158</y>
    </hint>
    <hint type="destinationlabel">
     <x>355</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_10</sender>
   <signal>clicked()</signal>
   <receiver>Pyatnashki</receiver>
   <slot>clickPushButton()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>103</x>
     <y>139</y>
    </hint>
    <hint type="destinationlabel">
     <x>395</x>
     <y>160</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_11</sender>
   <signal>clicked()</signal>
   <receiver>Pyatnashki</receiver>
   <slot>clickPushButton()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>179</x>
     <y>168</y>
    </hint>
    <hint type="destinationlabel">
     <x>376</x>
     <y>180</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_12</sender>
   <signal>clicked()</signal>
   <receiver>Pyatnashki</receiver>
   <slot>clickPushButton()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>239</x>
     <y>174</y>
    </hint>
    <hint type="destinationlabel">
     <x>414</x>
     <y>201</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_13</sender>
   <signal>clicked()</signal>
   <receiver>Pyatnashki</receiver>
   <slot>clickPushButton()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>48</x>
     <y>211</y>
    </hint>
    <hint type="destinationlabel">
     <x>344</x>
     <y>218</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_14</sender>
   <signal>clicked()</signal>
   <receiver>Pyatnashki</receiver>
   <slot>clickPushButton()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>102</x>
     <y>200</y>
    </hint>
    <hint type="destinationlabel">
     <x>335</x>
     <y>211</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_15</sender>
   <signal>clicked()</signal>
   <receiver>Pyatnashki</receiver>
   <slot>clickPushButton()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>161</x>
     <y>194</y>
    </hint>
    <hint type="destinationlabel">
     <x>360</x>
     <y>188</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_16</sender>
   <signal>clicked()</signal>
   <receiver>Pyatnashki</receiver>
   <slot>clickPushButton()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>225</x>
     <y>229</y>
    </hint>
    <hint type="destinationlabel">
     <x>387</x>
     <y>234</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
